documentation/asciidoc/topics/ref_datagrid_service_resources.adoc

```diff
@@ -53,6 +53,13 @@ include::yaml/datagrid_service.yaml[]
 |`spec.configMapName`
 |Specifies a `ConfigMap` that contains {brandname} configuration.
 
+|`spec.configListener`
+|Creates a `listener` pod in each {brandname} cluster that allows {ispn_operator} to reconcile server-side modifications with `Infinispan` resources such as the `Cache` CR.
+
+The `listener` pod consumes minimal resources and is enabled by default.
+Setting a value of `false` removes the `listener` pod and disables reconciliation.
+You should do this only if you access `Infinispan` resources through the {brandname} Console, CLI, or client applications.
```

Comment:
I think this can be made a little clearer, how about:

"You should do this only if you do not require declarative Kubernetes representations of `Infinispan` resources created via the {brandname} Console, CLI, or client applications."

Also, should `Infinispan` be parametised for downstream?

Reply:
Thanks. I've changed `Infinispan` resources to `{brandname}` resources for downstream.